examples/generative/corrdiff/conf/config_train.yaml

```diff
@@ -26,6 +26,7 @@ arch: ddpmpp-cwb-v0-regression
   # Network architecture [ddpmpp, ddpmpp-cwb-v2, ncsnpp, adm, ddpmpp-cwb-v0-regression]
 precond: unetregression
   # Preconditioning & loss function [vp, ve, edm, unetregression, resloss]
+task: regression
```

Comment:
Might be good to document the other options available here (similar to other configs)

Reply:
Done